vowpalwabbit/parse_regressor.cc

```diff
@@ -291,15 +291,15 @@ void save_load_header(
 
           if (!read)
           {
-            msg << all.triples[i] << " ";
-            memcpy(triple, all.triples[i].c_str(), 3);
+            msg << all.triples[i].data() << " ";
```

Comment:
Are you sure this will do what you want it to do? Is this vector null terminated?

Its dangerous to treat a container's data() section as a null-terminated string. Please don't do that